views/layouts/sidebar_page.tpl.php

```diff
@@ -7,6 +7,7 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <link href="/css/style.css" rel="stylesheet" />
```

Comment:
Hmm, this line seems extra and not related to your work, it seems.

Reply:
yes, but the menu isnțt horizontal on admin page, if I don't include the css file

Reply:
Ok to stay in this case.